astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -646,6 +646,29 @@ def test_read_complex():
         Table.read(txt, format='ascii.ecsv')
 
 
+@pytest.mark.skipif(np.__version__.startswith('1.18.'),
```

Comment:
Use this:

https://github.com/astropy/astropy/blob/b49ad06b4de9577648a55d499d914e08baeef2c6/astropy/utils/compat/numpycompat.py#L16

```suggestion
@pytest.mark.skipif(NUMPY_LT_1_19,
```